website/content/docs/v1.0/Guides/deploying-cilium.md

```diff
@@ -128,23 +128,8 @@ As such you should definitely not host this somewhere publicly accessible.
 
 ## Method 4: Helm manifests inline install
 
+This method requires that you have set your CNI to `none` as outlined [above](#machine-config-preparation).\
```

Comment:
@smira does gridsome render this?

Reply:
no idea really, but you can `cd website; gridsome develop` and verify it

Reply:
verified that it works